browser-extension/manifest.json

```diff
@@ -27,6 +27,18 @@
         "default": "Ctrl+Shift+Y"
       }
     }
-  }
+  },
+
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "dlalic@users.noreply.github.com",
```

Comment:
The id is required to link the web extension to the native messaging binary. Since I didn't want to use my email, used the github's no reply foo. Idk if it has to be an email tho, think I tried "foo", it didn't work, didn't want to waste time to check why so I put the email in.

Still not sure about the whole native messaging thing, seems like an overkill. Templating + web assembly seems like a saner approach.